README.md

```diff
@@ -29,7 +29,7 @@ import App from './components/App';
 ReactDOM.render(
   <SipProvider
     host="sip.example.com"
-    suffix="ws" // This value is appended to host string for websockets (e.g. wss://server:7443/ws); null by default
+    path="ws" // Path in socket URI (e.g. wss://sip.example.com:7443/ws); null bu default
```

Comment:
Sorry, I made a typo in `by` – could you please fix? After renaming is done in the code, should look good to me. No need to squash, thanks!

Reply:
...and I didn't bother to read what I'm copying ;)